tests/routes/desktop_browser/test_crypto.py

```
@@ -0,0 +1,49 @@
+"""Tests for browser-cookie key derivation (Argon2id)."""
+from __future__ import annotations
+
+import pytest
+
+
+class TestDeriveCookieKey:
+    def test_returns_64_char_hex_string(self):
+        from tinyagentos.routes.desktop_browser.crypto import derive_cookie_key
+
+        key = derive_cookie_key(password="hunter2", user_salt=b"u" * 16)
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "password"

(S106)